webapp/src/components/SmartTagsTable.tsx

```diff
@@ -0,0 +1,476 @@
+                                cell.outcomeCount[outcome] === 1 ? "" : "s"
+                              } are `}
+                              <Typography
+                                variant="inherit"
+                                component="span"
+                                fontWeight="bold"
+                              >
+                                {OUTCOME_PRETTY_NAMES[outcome]}
+                              </Typography>
+                            </Typography>
+                            <Typography variant="inherit">
+                              {`out of ${cell.utteranceCount} utterance${
+                                cell.utteranceCount === 1 ? "" : "s"
+                              } tagged `}
+                              <Typography
+                                variant="inherit"
+                                component="span"
+                                fontWeight="bold"
+                              >
+                                {SMART_TAG_FAMILY_PRETTY_NAMES[family]}
+                              </Typography>
+                            </Typography>
+                            <Typography variant="inherit">
+                              {`out of ${row.utteranceCount} utterance${
+                                row.utteranceCount === 1 ? "" : "s"
+                              } labeled `}
+                              <Typography
+                                variant="inherit"
+                                component="span"
+                                fontWeight="bold"
+                              >
+                                {row.filterValue}
+                              </Typography>
+                            </Typography>
+                          </>
+                        }
+                      >
+                        <Box
+                          component={motion.div}
+                          bgcolor={(theme) =>
+                            theme.palette[OUTCOME_COLOR[outcome]].main
+                          }
+                          height="100%"
+                          animate={{
+                            width: `${
+                              row.utteranceCount &&
+                              (100 * cell.outcomeCount[outcome]) /
+                                row.utteranceCount
+                            }%`,
+                          }}
+                          initial={false}
+                          transition={{ type: "tween" }}
+                          display="flex"
+                        >
+                          <Link
+                            component={RouterLink}
+                            width="100%"
+                            to={`/${jobId}/dataset_splits/${datasetSplitName}/performance_overview${constructSearchString(
+                              {
+                                label: [row.filterValue],
```

Comment:
@lindsaydbrin
> if you are looking at Predictions instead of Labels, the link still goes to the exploration space with the Label filtered by the class in question
Good catch! Thank you very much! That will be the fix:
```suggestion
                                [selectedMetricPerFilterOption]: [
                                  row.filterValue,
                                ],
```

Reply:
Awesome, so then in theory it's robust if we come up with more options for row labels! :)

Reply:
Ho yeas! 🤟